### PR TITLE
Document DICOM support in README

### LINKED ISSUE #215: Update documentation to include DICOM rules explanations
This should include what kinds of redaction are currently supported and which will be supported in the future (e.g. deletion of metadata tags is available now, but UID mapping is currently not available.

### PR DESCRIPTION
**Pull request by naglepuff**
Fix #215

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,8 +127,12 @@ Use the additional properties:
 * `expected_count` (optional): if the piece of metadata can contain multiple values, specify how many are expected using this property. Defaults to `1`. If the `expected_type` is `rational`, this should be the expected number of rationals. That is, an `expected_count` of 1 would match with 2 integer values in the metadata.
 
 ### Supported Formats
-Currently, `imagedephi` supports redaction of pure `tiff` files as well as Aperio (.svs) files.
+Currently, `imagedephi` supports redaction of the following types of files:
+* TIFF
+* Aperio (a tiff-like format, typically uses the extension `.svs`
```
Comment (manthey) [inline README.md:132]: This needs a final `)`

File: README.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ For whole slide image formats based on the tiff standard, `imagedephi` allows a
 
 
 ## Ruleset Format Overview
-In order to read the base rules and build your own custom rule sets, it is important to understand the format in which rulesets are specified. Rulesets are defined by `.yaml` files (one ruleset per file), and are a dictionary with the following top-level tags: `name`, `description`, `output_file_name`, `tiff`, and `svs`.
+In order to read the base rules and build your own custom rule sets, it is important to understand the format in which rulesets are specified. Rulesets are defined by `.yaml` files (one ruleset per file), and are a dictionary with the following top-level tags: `name`, `description`, `output_file_name`, `tiff`, `svs` and `dicom`.
```
Comment (manthey) [inline README.md:37]: We've lost the Oxford comma.  Elsewhere we use the Oxford comma.

Comment (naglepuff) [reply README.md]: Fixed in 8101650

File: README.md (hunk 1 above)
Comment (naglepuff) [reply README.md]: Fixed in 8101650

**Review by manthey — APPROVED** [review]

**Pull request merged**